packages/@aws-cdk/aws-stepfunctions-tasks/lib/eks/create-cluster.ts

```diff
@@ -0,0 +1,225 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/** Properties for creating a cluster with EksCreateCluster */
+export interface EksCreateClusterProps extends sfn.TaskStateBaseProps {
+  /** Name of the cluster */
+  readonly name: string;
+
+  /**
+   * The desired Kubernetes version for your cluster.
+   * @default - latest
+   */
+  readonly kubernetesVersion?: string;
+
+  /** EKS Role ARN to create a cluster */
+  readonly role: string;
+
+  /** VPC configuration used by the cluster control plane */
+  readonly resourcesVpcConfig: VpcConfigRequest;
+
+  /**
+   * Enable or disable exporting the Kubernetes control plane logs for your cluster to CloudWatch Logs.
+   * @default - no logging configuration
+   */
+  readonly loggingConfiguration?: LoggingConfig;
```

Comment:
by convention for the most part the types for configurable properties are suffixed `Options` - i.e. `LoggingOptions`